doc/user/content/sql/show-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "SHOW CONNECTIONS"
+description: "`SHOW CONNECTIONS` lists the connections configured in Materialize."
+menu:
+  main:
+    parent: commands
+aliases:
+    - /sql/show-connection
+---
+
+`SHOW CONNECTIONS` lists the connections configured in Materialize.
+
+## Syntax
+
+{{< diagram "show-connections.svg" >}}
+
+Field                | Use
+---------------------|-----
+_schema&lowbar;name_ | The schema to show connections from. If omitted, connections from all schemas are shown. For available schemas, see [`SHOW SCHEMAS`](../show-schemas).
```

Comment:
The bnf lists `FULL` and `EXTENDED`. Should we document those here too?

Reply:
Yeah, I'm adding these in now that it's clear that connections will always be user-defined. This is and `LIKE`/`WHERE` are really inconsistently documented in other pages, so I'll patch the rest as a follow-up.